CodeStarterCamp_Week4/main.swift

```diff
@@ -8,5 +8,110 @@
 
 import Foundation
 
-print("Hello, World!")
+class BodyCondition {
+    var upperBodyStrength: Int {
+        
+        didSet(oldValue) {
+            print("상체근력이 \(upperBodyStrength - oldValue) 상승합니다.")
+        }
+    }
+    var lowerBodyStrength: Int {
+        
+        didSet(oldValue) {
+            print("하체근력이 \(lowerBodyStrength - oldValue) 상승합니다.")
+        }
+    }
+    var muscularEndurance: Int {
+        
+        didSet(oldValue) {
+            print("근지구력이 \(muscularEndurance - oldValue) 상승합니다.")
+        }
+    }
+    var fatigueDegree: Int {
+        
+        didSet(oldValue) {
+            if fatigueDegree > oldValue {
+                print("피로도가 \(fatigueDegree - oldValue) 상승합니다.")
+            } else {
+                print("피로도가 \(oldValue - fatigueDegree) 하락합니다.")
+            }
+        }
+    }
+    var coreStrength: Int {
+        
+        didSet(oldValue) {
+            print("코어힘이 \(coreStrength - oldValue) 상승합니다.")
+        }
+    }
+    var bodyWeight: Int = 100 {
+        
+        willSet(afterDiet) {
+            print("\(bodyWeight)kg 에서 \(afterDiet-bodyWeight)kg 감량했습니다.")
+        }
+    }
+    init(upperBodyStrength: Int, lowerBodyStrength: Int, muscularEndurance : Int, fatigueDegree: Int, coreStrength: Int) {
+        self.upperBodyStrength = upperBodyStrength
+        self.lowerBodyStrength = lowerBodyStrength
+        self.muscularEndurance = muscularEndurance
+        self.fatigueDegree = fatigueDegree
+        self.coreStrength = coreStrength
+    }
```

Comment:
```suggestion
    init(
        upperBodyStrength: Int,
        lowerBodyStrength: Int,
        muscularEndurance : Int,
        fatigueDegree: Int,
        coreStrength: Int
     ) {
        self.upperBodyStrength = upperBodyStrength
        self.lowerBodyStrength = lowerBodyStrength
        self.muscularEndurance = muscularEndurance
        self.fatigueDegree = fatigueDegree
        self.coreStrength = coreStrength
    }
```

파라미터가 많을 경우, 이렇게 파라미터에 줄바꿈을 적용해주기도 합니다. 참고만 해주세요 !